pkg/generate/code/set_sdk_test.go

```diff
@@ -1778,3 +1778,28 @@ func TestSetSDK_MQ_Broker_Create(t *testing.T) {
 		code.SetSDK(crd.Config(), crd, model.OpTypeCreate, "r.ko", "res", 1),
 	)
 }
+
+func TestSetSDK_EC2_VPC_ReadMany(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+
+	g := testutil.NewModelForService(t, "ec2")
+
+	crd := testutil.GetCRDByName(t, g, "Vpc")
+	require.NotNil(crd)
+
+	expected := `
+	if r.ko.Spec.DryRun != nil {
```

Comment:
nit: wasn't dry-run supposed to be excluded from the VPC CR? It may just be unit tests but better to keep it consistent with actual controller. : )

Reply:
That's correct. Does that mean service controller's _generator.yaml_ needs to stay in sync with https://github.com/aws-controllers-k8s/code-generator/blob/main/pkg/testdata/models/apis/ec2/0000-00-00/generator.yaml ?

Reply:
not necessarily that's why I mentioned nit.
I mentioned this comment just because why unit test something(dry-run here) which is never actually used in reality.